ChangeLog.md

```diff
@@ -1,5 +1,193 @@
 # Changelog for haskell-language-server
 
+## 1.0.0
+
+This is the celebratory release of Haskell Language Server 1.0.0!
+This release includes a lot of internal changes, bug fixes, leaks plugged, and performance improvements, thanks to all our contributors.
+Among others,
+- We added the support for GHC 8.10.4, and removed the support for GHC 8.10.1
```

Comment:
Can we mention the hie-bios changes as @jneira pointed out?

Reply:
Oops, I missed that. Thank you for pointing it out.